spec/models/searchworks_item_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+describe SearchworksItem do
+  let(:item_id) { SearchworksItem.new }
```

Comment:
RSpec will automatically instantiate the class w/o any arguments as an implicit subject.  You can change that by defining your own subject.